docker-compose.yml

```diff
@@ -14,11 +14,12 @@ services:
     command:
       - |
         pulumi login --local
+        export TRACING_BACKEND_URL="http://host.docker.internal:9411/api/v1/spans"
```

Comment:
Tiny nitpick: I don't think `export` is required here; the variable doesn't need to be available to subprocesses, since you're passing it directly to the process via the command line option.